Formula/jboss-forge.rb

```diff
@@ -1,9 +1,9 @@
 class JbossForge < Formula
   desc "Tools to help set up and configure a project"
   homepage "https://forge.jboss.org/"
-  url "https://downloads.jboss.org/forge/releases/3.5.1.Final/forge-distribution-3.5.1.Final-offline.zip"
-  version "3.5.1.Final"
-  sha256 "13121caf9c103f87d9344dff717cab52c383614acebfbe117e40ef5c094c36ba"
+  url "https://downloads.jboss.org/forge/releases/3.6.0.Final/forge-distribution-3.6.0.Final-offline.zip"
+  version "3.6.0.Final"
+  sha256 "e7093b7beef94031d00c771306aeccf69a15d45029de92bef51d8b3c3ab3af9d"
 
   bottle :unneeded
 
```

Comment:
After the bottle line, this needs a `depends_on :java => "1.8+"` as it requires a Java 8+ JDK.